tests/Calendar/CalendarTest.php

```diff
@@ -1,8 +1,9 @@
 <?php
 
+use PHPUnit\Framework\TestCase;
```

Comment:
Any reason to switch to namespaced PHPUnic class names? Since we are using PHPUnit 4.8, those namespaced classes are just an alias of the non-namespaced classes...

Reply:
I think we should add different PHPUnit versions for the different PHP versions.
And that's the reason why using the class-based PHPUnit namespace.

Reply:
> I think we should add different PHPUnit versions for the different PHP versions.

Why?

Reply:
According to the official [website](https://phpunit.de/index.html), we can see the PHPUnit version supports different PHP versions.